querybook/webapp/const/metastore.ts

```diff
@@ -116,6 +119,9 @@ export interface IDataColumn {
     name: string;
     table_id: number;
     type: string;
+    stats?: ITableColumnStats[];
```

Comment:
instead of stats?, can you separate the type out to be IDetailedDataColumn extends IDataColumn?